src/lib/pickles/test/optional_custom_gates/test_fix_domains.ml

```diff
@@ -8,8 +8,15 @@
 
 open Kimchi_backend_common.Plonk_constraint_system.Plonk_constraint
 
-let add_constraint c =
-  Pickles.Impls.Step.assert_ { basic = T c; annotation = None }
+open Pickles.Impls.Step
+
+let add_constraint c = assert_ { basic = T c; annotation = None }
+
+let add_plonk_constraint c =
+  assert_
+    { basic = Kimchi_backend_common.Plonk_constraint_system.Plonk_constraint.T c
```

Comment:
Does this line need to be so long even when `Kimchi_backend_common.Plonk_constraint_system.Plonk_constraint` has been opened before? I wonder if this could have a shorter name, but okay this only appears one in the file... so might not be worth it

Reply:
True, lemme remove it